### DISCUSSION TITLE
Welcome to auto-task Discussions!

### DISCUSSION THREAD

**Discussion by HCLonely** [HCLonely/auto-task-old#37] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stickerx** [comment]
Is This Real if I Remove Games from The Whitelist Will I Get Banned Forever

https://giveaway.su/

![2021-10-14_195046](https://user-images.githubusercontent.com/41187653/137379890-6209300f-aaa4-492e-840a-4313e998d195.jpg)

**Reply by HCLonely** [reply]
> Is This Real if I Remove Games from The Whitelist Will I Get Banned Forever
> 
> [giveaway.su](https://giveaway.su/)
> 
> ![2021-10-14_195046](https://user-images.githubusercontent.com/41187653/137379890-6209300f-aaa4-492e-840a-4313e998d195.jpg)

REAL!

**Reply by stickerx** [reply]
OMG

**Comment by stickerx** [comment]
repage function of Giveaway was useful why canceled ?

**Reply by HCLonely** [reply]
Because I got the "0000" key when I used the "repage" function, if you want to continue to use this function, please add the following to the code on line 9707:
```
        repage: {
          show: true,
          text: 'RePage',
          title: getI18n('RePage')
        }
```

**Reply by stickerx** [reply]
I should follow the leader

after adding the to the code  "repage"
![2021-11-03_153807](https://user-images.githubusercontent.com/41187653/140081613-d9e6dbef-25ae-4a32-bded-83f534c5cade.png)

I remove him again thank you for noticing

**Comment by stickerx** [comment]
@HCLonely 
Hi :)

please is there a way to bypass **Disable AdBlock on our website** without disabling AdBlock & Antivirus

this task is annoying 


![2021-11-30_191646](https://user-images.githubusercontent.com/41187653/144105219-dec14b5c-019c-4d43-84c2-035be9e136ef.jpg)

**Reply by HCLonely** [reply]
```
$('tr[data-action-id="adjs"]').html('<td><div class="label label-danger" data-toggle="tooltip" data-placement="right" title="Disable AdBlock on our website"><i class="fa fa-ban"></i></div></td><td><a href="#adblock-modal">Disable AdBlock on our website</a></td><td class="text-right"><button type="button" class="btn btn-xs btn-default btn-success" data-type="action.adjs" data-action="adjs"><i class="glyphicon glyphicon-refresh"></i></button></td>');
$('.glyphicon.glyphicon-refresh').parent('a:not(.btn-success)').each(function (i) {
  const href = $(this).attr('href');
  $(this).removeAttr('href');
  this.click();
  $(this).attr('href', href);
});
```

**Reply by stickerx** [reply]
Thank you so much
I'm sorry, But I'm a newbie at coding
Where i should put this code?

I am talking about  givee-club
![210x60-header-en](https://user-images.githubusercontent.com/41187653/144278840-f3b5efd3-53e7-45b5-baec-1bacb25262ea.png)

**Reply by HCLonely** [reply]
添加以下过滤器到你的广告屏蔽插件中
`@@||givee.club/adsense.js$script`

**Reply by stickerx** [reply]
@HCLonely
Thank you so much

Does this also work like this?

@@||key-hub.eu/adsense.js$script

**Reply by HCLonely** [reply]
It doesn't work
